packages/Form/Input/checkbox/src/checkbox.scss

```diff
@@ -107,7 +108,7 @@
     .af-form__input-checkbox {
       position: absolute;
       z-index: -1;
-      visibility: hidden;
+      visibility: inherit;
```

Comment:
A la place de inherit est ce qu'il ne serait pas mieux de l'enlever ?

Reply:
done and it works well @arnaudforaison

Reply:
finalement, tu as enlevé le "visibility: hidden;" ou tu l'as remplacé par "visibility: inherit" ?
Si tu le supprimes complètement, il n'y a pas d'effet de bord sur les autres types de checkbox ?

Reply:
Il a été supprimé.
@antoine-blancke Est ce qu'il y a des effets de bord ?